src/sidebar/test/oauth-auth-test.js

```diff
@@ -252,45 +267,14 @@ describe('sidebar.oauth-auth', function () {
 
       return callTokenGetter()
         .then(resetHttpSpy)
-        .then(expireAccessToken)
+        .then(expireAndRefreshAccessToken)
         .then(assertRefreshTokenWasUsed('firstRefreshToken'))
         .then(resetHttpSpy)
         .then(assertThatTokenGetterNowReturnsNewAccessToken)
-        .then(expireAccessToken)
+        .then(expireAndRefreshAccessToken)
         .then(assertRefreshTokenWasUsed('secondRefreshToken'));
     });
 
-    // While a refresh token HTTP request is in-flight, calls to tokenGetter()
-    // should just return the old access token immediately.
-    it('returns the access token while a refresh is in-flight', function() {
```

Comment:
This test was removed because the behaviour of `tokenGetter()` was changed to wait for the refreshed token when a refresh is happening, which is simpler to reason about IMO.